Directory.Build.props

```diff
@@ -1,4 +1,4 @@
-<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
```

Comment:
This says "LICENSE" but the license is called "LICENSE.md" - can we make one of them in sync?

Reply:
This is equal to what's on roslyn and runtime where they have License.txt instead, should we change our file from md to txt?

Reply:
Well they are both wrong too, this is a copy and paste error, it used to be called "LICENSE".

Reply:
https://github.com/dotnet/project-system/commit/167b0b83c8108df757989f15b03a8db9a795efd4

Reply:
I'll change the file name in the comments then and file an issue with the other repos so they do the same. Just to understand the context, why'd we switch to .md?

Reply:
Cos it looked better.

Reply:
@dmonroym I'm fixing this.